frontend/src/mixins/isChangeSelectedWebsite.js

```diff
@@ -0,0 +1,19 @@
+import store from "../store";
+
+export const isChangeSelectedWebsite = {
+    mounted() {
+        store.watch(
```

Comment:
Define watcher with component, but not with store. Mixins allows you to define exact same properties as in regular component